UI/BindingSources/CSHARP/Binding/MainViewModel.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System.Collections.ObjectModel;
+using System.ComponentModel;
+
+namespace Binding;
+
+public class MainViewModel : INotifyPropertyChanged
```

Comment:
If we're going to do this go ahead and add Mvvm as a Feature... let's use the code gen to clean this up.